node/src/utils/external.rs

```diff
@@ -22,12 +22,10 @@ use crate::{
     tls,
 };
 
+/// Path to bundled resources.
 #[cfg(test)]
-lazy_static::lazy_static! {
-    /// Path to bundled resources.
-    pub static ref RESOURCES_PATH: PathBuf =
-        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../resources");
-}
+pub static RESOURCES_PATH: once_cell::sync::Lazy<PathBuf> =
+    once_cell::sync::Lazy::new(|| PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../resources"));
```

Comment:
You can import `once_cell::sync::Lazy` for `#[cfg(test)]` only.

Reply:
Imported using `#[cfg(test)]`